2021-offseason/src/main/java/frc/robot/OI.java

```diff
@@ -1,5 +1,15 @@
+package frc.robot;
+
 import edu.wpi.first.wpilibj.XboxController;
-import edu.wpi.first.wpilibj.controller.*;
-public class OI {
+import edu.wpi.first.wpilibj.GenericHID.Hand;
 
-}
\ No newline at end of file
+
+public class OI {
+    public static double getTriggers(XboxController controller){
```

Comment:
spaces in this file too

Reply:
add a method comment imo, `getTriggers` doesn't obviously subtract the left hand from right hand inputs

Reply:
Or you could additionally name this function better (e.g. getTriggerTotal)